src/dfx/assets/new_project_motoko_files/README.md

```diff
@@ -46,6 +54,7 @@ Which will start a server at `http://localhost:8080`, proxying API requests to t
 
 If you are hosting frontend code somewhere without using DFX, you may need to make one of the following adjustments to ensure your project does not fetch the root key in production:
 
-- set`NODE_ENV` to `production` if you are using Webpack
-- use your own preferred method to replace `process.env.NODE_ENV` in the autogenerated declarations
+- set`DFX_NETWORK` to `production` if you are using Webpack
```

Comment:
Maybe we should reconsider this, because `DFX_NETWORK` will usually be `ic` for the production case (see index.js.hbs line 25, and builders/mod.rs line 334)

Reply:
Oh, that we can just say that. it's accurate
```suggestion
- set`DFX_NETWORK` to `ic` if you are using Webpack
```